.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: release-pipeline
+
+on:
+  release:
+    types:
+      - created
+
+
+jobs:
+  deploy-live:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Check HTML
+      uses: chabad360/htmlproofer@master
+      with:
+        directory: "."
+    - name: Deploy preview site
+    - uses: jakejarvis/s3-sync-action@master
+      with:
+        args: --exclude '*' --include '*.html' --include 'assets/*' --include 'global'
+      env:
+        AWS_S3_BUCKET: rtlive.de
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: 'eu-central-1'
+        SOURCE_DIR: '.'
+        DEST_DIR: ''
```

Comment:
This I don't get. What is this empty DEST_DIR?

Reply:
The empty DEST_DIR is the root level of the S3 bucket where the website lives

Reply:
If I see it right in the documentation, you can skip the argument in that case.

Reply:
True. I just wanted to make it clear that it's intentional to upload to the root of the bucket.